doc/setup.md

```diff
@@ -149,7 +149,41 @@ These instructions assume that you have [Homebrew](http://brew.sh/) installed.
 	```
 	sudo apt install graphviz-dev graphviz
 	```
+## Configuring Shell Environment for Development
 
+When developing with SkyPortal, you may find it helpful to also configure your shell enviroment. Here is how you can do that by adding certain configurations to your `.zshrc` file.
```

Comment:
These are instructions for mac only, so you should move them to that section of the docs. Also, don't forget to provide the homebrew command to install hdf5, c-bloscc, ... and mention that the user should copy the paths homebrew printed to put them in the .zshrc file